tests/contrib/botocore/test.py

```diff
@@ -1013,6 +1014,7 @@ def test_sns_send_message_trace_injection_with_no_message_attributes(self):
 
     @mock_sns
     @mock_sqs
+    @pytest.mark.xfail(strict=False)  # FIXME: flaky test
```

Comment:
how does `xfail(strict=False)` work?

we still run the best, but we just don't care about the results?

- No `xfail()`: it must pass
- `xfail()`: it must fail
- `xfail(strict=False)`: it is run, but the result doesn't matter
- `xskip()`: don't run the test

?

Reply:
https://docs.pytest.org/en/6.2.x/flaky.html#xfail-strict

Yeah run it and report the failure but don't fail the run.